src/lib/components/chat/ChatMessages.svelte

```diff
@@ -54,7 +54,7 @@
 				>
 					{#if $page.data?.assistant.avatar}
 						<img
-							src="{base}/settings/assistants/{$page.data?.assistant._id.toString()}/avatar?hash=${$page
+							src="{base}/settings/assistants/{$page.data?.assistant._id.toString()}/avatar.jpg?hash=${$page
 								.data?.assistant.avatar}"
```

Comment:
```suggestion
								.data.assistant.avatar}"
```

because of the check above `{#if $page.data?.assistant.avatar}`, this should work ?